src/lib/containers/QueryWrapper.js

```diff
@@ -60,7 +62,7 @@ export default class QueryWrapper extends React.Component {
 
         if (prevProps.initQueryRequest.query.sql !== this.props.initQueryRequest.query.sql) {
             this.setState({isChecked: []})
-            this.executeQueryRequest(this.props.initQueryRequest, false)
+            this.executeQueryRequest(null, true)
```

Comment:
I understand that you are resetting the request in this case, but why do you now send null instead of the initial query request?